core/services/directrequestocr/drlistener.go

```diff
@@ -23,6 +25,26 @@ import (
 var (
 	_ log.Listener   = &DRListener{}
 	_ job.ServiceCtx = &DRListener{}
+
+	promOracleEvents = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_oracle_events",
+		Help: "Metric to track oracle events",
+	}, []string{"oracle", "event"})
+
+	promRequestErrors = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
How about we make a bit more specific counters (and with common prefixes):
- functions_request_computation_success
- functions_request_computation_error
- functions_request_internal_error
- functions_request_timeout
- functions_request_confirmed

Reply:
Yeah this naming pattern makes sense, will do. Thanks!

Reply:
I do agree that would be great. Will see what best we can do prior to the event.

Reply:
Done